OrbitQt/orbitlivefunctions.h

```diff
@@ -29,7 +29,7 @@ class OrbitLiveFunctions : public QWidget {
   void Initialize(SelectionType selection_type, FontType font_type, bool is_main_instance = true);
   void Refresh();
   void OnDataChanged();
-  void onRowSelected(int row);
+  void onRowSelected(std::optional<int> row);
```

Comment:
If you're touching this method, can you change it to follow our coding style (`OnRowSelected`)? Thanks!
(I think this isn't overwritten from `QWidget`, right? If this is the case, add `override` please)

Reply:
Thanks for the suggestion! I will update the name accordingly. Also, you are right. I just checked that it is not overwritten from `QWidget`.